### PR TITLE
Implement db seeding script

### PR DESCRIPTION
**Pull request by KJou**
@pxshelly, could you take a look at my seeding script for the shared product data?

https://trello.com/c/eBYmkilR/5-implement-seeding-script

### CODE REVIEW

File: database/seedData.js [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
-mongoose.connect('mongodb://localhost/n3rdstorm', (err, db) => {
-  if (err) {
-    console.error(err);
-  }
-  db.dropDatabase();
-  db.close();
-});
-
-mongoose.connect('mongodb://localhost/n3rdstorm');
+mongoose.connect('mongodb://localhost/n3rdstorm', () => mongoose.co);
```
Comment (pxshelly) [inline database/seedData.js]: What does mongoose.co do in the callback?

File: database/seedData.js [hunk 2]
```diff
@@ -79,18 +73,15 @@ const sharedProduct5 = new ReviewSummary({
...
-});
+const savePromises = [
+  sharedProduct1.save(),
+  sharedProduct2.save(),
+  sharedProduct3.save(),
+  sharedProduct4.save(),
+  sharedProduct5.save(),
+];
+
+db.dropDatabase()
```
Comment (pxshelly) [inline database/seedData.js]: Does this database need to be dropped? It seems that the database is being seeded and then immediately dropped afterwards.

**Review by pxshelly — COMMENTED** [review]
It may be a good idea to place the connection to Mongo and the schema definition in its own 'index.js'. This way you can import the connection into your seed script. By doing so, this can reduce code redundancy when it's time to connect to your database for your service.

File: database/seedData.js (hunk 1 above)
Comment (KJou) [reply database/seedData.js]: Oops, I mistakenly left that in there. Removed now.

File: database/seedData.js (hunk 2 above)
Comment (KJou) [reply database/seedData.js]: The purpose of this is to drop whatever still exists in the DB before saving new data. Drop first, then seed.

**Comment by KJou** [comment]
@pxshelly, split the db connection into its own file based upon your suggestion. Is that what you meant?

@jessicasm, any thoughts or comments?

**Review by pxshelly — APPROVED** [review]
Looks good!

**Comment by jessicasm** [comment]
@KJou Looks good to me!

**Pull request merged**